proposals/lip-0071.md

```diff
@@ -28,20 +28,20 @@ In the current [Lisk protocol][lip-0042], the rewards are the same for all activ
 
 ### Minimum Reward for Active Validators
 
-This module assigns a minimum reward i.e. `minimumRewardActiveValidators` per round to all active validators to cover the minimal operational cost to run a validator. For the Lisk mainchain, the minimum reward is 10<sup>7</sup> beddows or `0.1 LSK`.
+This module assigns a minimum reward i.e. `getMinimumRewardActiveValidators(blockHeader.height)` per round to all active validators to cover the minimal operational cost to run a validator. The minimum reward is 10% of the total reward (`getNumberActiveValidators() * getDefaultRewardAtHeight(blockHeader.height)`) in a round at the current block height. For the Lisk mainchain (assuming `101` active validators), the minimum reward is 10<sup>7</sup> beddows or `0.1 LSK` at current block height.
```

Comment:
I would erase the parenthesis (`getNumberActiveValidators() * getDefaultRewardAtHeight(blockHeader.height)`) because I think is a bit confusing.